mailbox/postgres/src/test/java/org/apache/james/mailbox/jpa/mail/dao/PostgresMailboxDAOTest.java

```diff
@@ -19,28 +19,31 @@
 
 package org.apache.james.mailbox.jpa.mail.dao;
 
-import static org.assertj.core.api.Assertions.as;
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.util.List;
 import java.util.Set;
 
+import org.apache.james.backends.postgres.PostgresExtension;
 import org.apache.james.core.Username;
 import org.apache.james.mailbox.jpa.PostgresMailboxId;
+import org.apache.james.mailbox.jpa.mail.PostgresMailboxModule;
 import org.apache.james.mailbox.model.Mailbox;
 import org.apache.james.mailbox.model.MailboxId;
 import org.apache.james.mailbox.model.MailboxPath;
 import org.apache.james.mailbox.model.UidValidity;
 import org.apache.james.mailbox.model.search.ExactName;
 import org.apache.james.mailbox.model.search.MailboxQuery;
 import org.apache.james.mailbox.model.search.PrefixedWildcard;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
 import com.google.common.collect.ImmutableSet;
 
 import reactor.core.publisher.Mono;
 
-public abstract class PostgresMailboxDAOTest {
+public class PostgresMailboxDAOTest {
```

Comment:
I think @chibenwa was saying to remove that test class, cause the test for MailboxMapper should be enough to cover the DAO underneath?

Reply:
I remember I have removed it :((

Reply:
deleted